terraform/aws/tests/launch.sh

```
@@ -22,8 +20,11 @@ ln -s -f ${binary_name} ${BINARY}
 
 mkdir -p ${LOG_DIR}
 
-cmd="./${BINARY} ${FLAGS}"
-time ${cmd} > ${LOG_DIR}/${BINARY}.STDOUT 2> ${LOG_DIR}/${BINARY}.STDERR < /dev/null
+# We ignore errors from here on. Failing tests are fine, and we still want
+# to create the DONE file.
+set +e
```

Unless there are a lot of commands that we want to ignore errors on, I prefer `|| true` instead of `set +e`.